bio/sexdeterrmine/wrapper.py

```
@@ -0,0 +1,44 @@
+from snakemake.shell import shell
+from tempfile import TemporaryDirectory
+
+import os
+import os.path
+
+log = snakemake.log_fmt_shell(
+    stdout=False,
+    stderr=True,
+    append=False,
+)
+extra = snakemake.params.get("extra", "")
```

Ruff (0.7.0)

Undefined name `snakemake`

(F821)